x-pack/plugins/lens/public/drag_drop/providers.tsx

```diff
@@ -176,7 +258,12 @@ export function ReorderProvider({
             </p>
             <p id={`lnsDragDrop-reorderInstructions-${id}`}>
               {i18n.translate('xpack.lens.dragDrop.reorderInstructions', {
-                defaultMessage: `Press space bar to start a drag. When dragging, use arrow keys to reorder. Press space bar again to finish.`,
+                defaultMessage: `Press space bar to start reordering the dimension group. When dragging, use arrow keys to reorder. Press space bar again to finish.`,
+              })}
+            </p>
+            <p id={`lnsDragDrop-groupMovementInstructions-${id}`}>
+              {i18n.translate('xpack.lens.dragDrop.groupMovementInstructions', {
+                defaultMessage: `Use right arrow and left key to move dimension to the next group.`,
```

Comment:
In addition to this `id` never matching up to any button, I couldn't get this to work on any field... Should it?

(Using Firefox on Mac)

Reply:
No, not yet – this is a preparation for some other functionalities, I forgot to delete this part.